tests/acceptance/modules/Users/UsersCest.php

```diff
@@ -50,7 +50,7 @@ public function testEmailSettingsMailAccountAdd(AcceptanceTester $I, UsersTester
         $I->fillField('email_user', 'testuser_name');
         $I->fillField('email_password', 'testuser_pass');
         $I->click('Test Settings');
-        $I->wait(10);
+        $I->wait(20);
```

Comment:
Is there no way to have the test suite wait until the AJAX call is finished (or the page is loaded, depending on what this is testing)?

Reply:
Good idea, should definitely be possible. Ideally I don't want a test that takes 20+ seconds.

Reply:
We had `sleep`s like this cause a lot of test slowdowns on a Rails app I worked on before, and I believe that was our solution in the end. Hopefully however SuiteCRM is integrating with selenium provides a way to do this :)